docs/conf.py

```diff
@@ -39,6 +39,11 @@
 autodoc_member_order = 'bysource'
 nitpicky = True
 nitpick_ignore = [
-    ('py:class', 'Evaluator')  # https://github.com/sphinx-doc/sphinx/issues/10785
+    ('py:class', 'Evaluator'),  # https://github.com/sphinx-doc/sphinx/issues/10785
+
+    # The Python documentation has unittest.loader.* and unittest.suite.* classes
+    # documented, but directly under "unittest".
+    ('py:class', 'unittest.loader.TestLoader'),
+    ('py:class', 'unittest.suite.TestSuite'),
```

Comment:
What happens without these?

Reply:
```
/Users/adam/Documents/repositories/sybil/sybil/sybil.py:docstring of sybil.sybil.Sybil.unittest:1: WARNING: py:class reference target not found: unittest.loader.TestLoader
/Users/adam/Documents/repositories/sybil/sybil/sybil.py:docstring of sybil.sybil.Sybil.unittest:1: WARNING: py:class reference target not found: unittest.suite.TestSuite
/Users/adam/Documents/repositories/sybil/sybil/sybil.py:docstring of sybil.sybil.Sybil.unittest:1: WARNING: py:class reference target not found: unittest.suite.TestSuite
/Users/adam/Documents/repositories/sybil/sybil/sybil.py:docstring of sybil.sybil.SybilCollection.unittest:1: WARNING: py:class reference target not found: unittest.loader.TestLoader
/Users/adam/Documents/repositories/sybil/sybil/sybil.py:docstring of sybil.sybil.SybilCollection.unittest:1: WARNING: py:class reference target not found: unittest.suite.TestSuite
/Users/adam/Documents/repositories/sybil/sybil/sybil.py:docstring of sybil.sybil.SybilCollection.unittest:1: WARNING: py:class reference target not found: unittest.suite.TestSuite
```

Reply:
Okay, but if we removed the type annotations using these, they wouldn't be needed, right?
(Sphinx has a number of issues around type annotations, IIRC, 😓 )

Reply:
Yes please, minimal to keep `mypy` happy as described below I think :-/